frontend/src/components/MeetingRoom/UnreadMessagesBadge/UnreadMessagesBadge.tsx

```diff
@@ -1,5 +1,5 @@
 import { Badge } from '@mui/material';
-import { ReactElement } from 'react';
+import { ForwardedRef, forwardRef, ReactElement } from 'react';
```

Comment:
Looks like `forwardRef` is deprecated: https://react.dev/reference/react/forwardRef.  Is that still OK for us to use?

Reply:
Nice catch! It's okay to leave as-is since we're using React v18 and that's a v19 feature. When we move to v19, it [looks like an easy fix](https://react.dev/blog/2024/12/05/react-19#ref-as-a-prop). (famous last words 🙈)

> we will be publishing a codemod to automatically update your components to use the new `ref` prop

🤞